decentraland/unity-explorer
# Chore: camera reel analytics and tilt

## What does this PR change?

- add all analytics, but not photo manipulation that comes from the context menu (delete, share, etc)
- add tilt behaviour (q/e) for the camera movement
- add controller support
- fixes placeApi exception when photo was taken outside of GenesisCity (on the map borders - sea view)
- sidebarUI adjustment by UX (@RominaMarchetti)

## How to test the changes?

1. Launch the explorer with respective CLI args
2. Enable InWorld Camera mode
3. Use q/e for tilting
4. Verify tilting is reset each time you lock/unlock mouse
5. Try to control camera with Gamepad (all gamepad buttons/controls are used, discover them). Gamepad aiming is only working when mouse is locked
6. Try to make picture on the border of the Genesis City - it should work as usual photo

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
